values-schema.yaml

```diff
@@ -207,6 +216,55 @@ definitions:
       tag:
         type: string
     title: Container image
+    type: object
+  job:
+    additionalProperties: false
+    properties:
+      enabled:
+        type: boolean
+      env:
+        $ref: '#/definitions/env'
+      image:
+        $ref: '#/definitions/image'
+      init:
+        properties:
+          image:
+            $ref: '#/definitions/image'
+          script:
+            $ref: '#/definitions/script'
+      mounts:
+        $ref: '#/definitions/mounts'
+      resources:
+        $ref: '#/definitions/resources'
+      schedule:
+        description: Must give a cron-type expression if the job-type is 'cronjob' (job.type).
+        type: string
+      script:
+        $ref: '#/definitions/script'
+      ttlSecondsAfterFinished:
+        default: 86400
+        pattern: ^[0-9]*$
+        type: number
+      type:
+        default: job
+        enum:
+          - job
+          - cronjob
+        type: string
+      volumes:
+        $ref: '#/definitions/volumes'
+    required:
+      - image
+      - script
+      - type
+  jobs:
+    description: You may pass an array of Job objects. The Kubernetes API accepts
```

Comment:
Would you mind adding a period to every description now that you started putting them at the end of yours? ;)

Reply:
I can also remove mine?

Reply:
When we have multiline descriptions it makes sense to use periods.